src/components/PrivateDataExplorer.js

```diff
@@ -103,7 +103,7 @@ export default _.flow(
           p([
             `Thank you for your interest in the Baseline Health Study data. Baseline data is currently only being shared
              with Baseline Health Study partner sites, Duke and Stanford. If you are a current researcher at one of those sites,
-             please reach out to your institutional contacts for information on how to obtain access. In the future, Baseline 
+             please reach out to your institutional contacts for information on how to obtain access. In the future, Baseline
```

Comment:
This looks like an unrelated change. Is it for linting?

Reply:
Sort of. My editor fixes trailing whitespace on save. The only reason it's not an actual lint error is because it's inside of a multiline string.